provider/cmd/pulumi-resource-cloudflare/schema.json

```diff
@@ -4611,7 +4760,7 @@
             "properties": {
                 "name": {
                     "type": "string",
-                    "description": "Name of the ruleset.\n"
+                    "description": "Name of the compression algorithm to use. Available values: `gzip`, `brotli`, `auto`, `default`, `none`\n"
                 }
```

Comment:
this might be a regression too?

Reply:
This looks like it's correct to me:
https://github.com/cloudflare/terraform-provider-cloudflare/blob/8008476152ba7f6e8d69dfc085a4dc2264c8f5b8/docs/resources/ruleset.md#L535-L540